### ISSUE TITLE
Roadie plugins seem to be pulling in very old dependencies

### ISSUE THREAD

**Issue by dtuite** [RoadieHQ/roadie-backstage-plugins#352]
## Expected Behavior

Roadie plugins should have up to date Backstage dependencies.

## Current Behavior

On the `master` branch of a clone of the `backstage/backstage` repo:

```
rm -rf node_modules

yarn install

ag 'backstage/core-components' ./**/package.json
```

Gives this output (I've omitted the output for non-Roadie plugins).

```
node_modules/@roadiehq/backstage-plugin-github-insights/node_modules/@backstage/core-app-api/package.json
34:    "@backstage/core-components": "^0.8.1",

node_modules/@roadiehq/backstage-plugin-github-insights/package.json
36:    "@backstage/core-components": "^0.8.0",

node_modules/@roadiehq/backstage-plugin-github-pull-requests/node_modules/@backstage/core-app-api/node_modules/@backstage/core-components/package.json
2:  "name": "@backstage/core-components",

node_modules/@roadiehq/backstage-plugin-github-pull-requests/node_modules/@backstage/core-app-api/package.json
34:    "@backstage/core-components": "^0.7.6",

node_modules/@roadiehq/backstage-plugin-github-pull-requests/node_modules/@backstage/core-components/package.json
2:  "name": "@backstage/core-components",

node_modules/@roadiehq/backstage-plugin-github-pull-requests/node_modules/@backstage/plugin-catalog-react/node_modules/@backstage/core-components/package.json
2:  "name": "@backstage/core-components",

node_modules/@roadiehq/backstage-plugin-github-pull-requests/node_modules/@backstage/plugin-catalog-react/package.json
35:    "@backstage/core-components": "^0.6.1",

node_modules/@roadiehq/backstage-plugin-github-pull-requests/package.json
33:    "@backstage/core-components": "^0.5.0",

node_modules/@roadiehq/backstage-plugin-travis-ci/node_modules/@backstage/core-app-api/node_modules/@backstage/core-components/package.json
2:  "name": "@backstage/core-components",

node_modules/@roadiehq/backstage-plugin-travis-ci/node_modules/@backstage/core-app-api/package.json
34:    "@backstage/core-components": "^0.7.6",

node_modules/@roadiehq/backstage-plugin-travis-ci/node_modules/@backstage/core-components/package.json
2:  "name": "@backstage/core-components",

node_modules/@roadiehq/backstage-plugin-travis-ci/node_modules/@backstage/plugin-catalog-react/node_modules/@backstage/core-components/package.json
2:  "name": "@backstage/core-components",

node_modules/@roadiehq/backstage-plugin-travis-ci/node_modules/@backstage/plugin-catalog-react/package.json
35:    "@backstage/core-components": "^0.4.2",

node_modules/@roadiehq/backstage-plugin-travis-ci/package.json
31:    "@backstage/core-components": "^0.3.0",
```

The latest version of `"@backstage/core-components"` is `0.8.2`, which is what should be present.

**Comment by Xantier** [comment]
https://github.com/backstage/backstage/pull/8724

**Comment by Xantier** [comment]
Closed with backstage/backstage#8724
